### DISCUSSION TITLE
Rebind the power key

### DISCUSSION THREAD

**Discussion by stepanzak** [YaLTeR/niri#716] [Ideas]
Hi, I'm aware of the option to disable the power button handling in Niri, but it would be great if we could rebind it - create a keybind with it just like with the function keys.
For example I would like to bind it to [wlogout](https://github.com/ArtsyMacaw/wlogout).

**Comment by YaLTeR** [comment]
Maybe you can already bind it if you disable niri power key handling and disable logind power key handling? Not sure

**Comment by Cure-Dev** [comment]
Yes, it works. By binding the XF86PowerOff key.

**Comment by Filiflo** [comment]
can somone tell me why this is the default? 
Not being able to turn off your pc with the power button or having it as a bind seems a bit wierd to me.
It just suddenly hibernating after installation is very surprising.

**Reply by jaschiu** [reply]
Agreed, I made https://github.com/niri-wm/niri/discussions/4132 discussing this
